ui/components/ui/editable-label/editable-label.js

```diff
@@ -83,8 +83,13 @@ export default class EditableLabel extends Component {
 
   renderReadonly() {
     return (
-      <Box display={DISPLAY.FLEX} alignItems={AlignItems.center} gap={3}>
-        <Text variant={TextVariant.bodyLgMedium}>{this.state.value}</Text>
+      <Box display={Display.Flex} alignItems={AlignItems.center} gap={3}>
+        <Text
+          variant={TextVariant.bodyLgMedium}
+          style={{ wordBreak: 'break-word' }}
```

Comment:
Does the provided `overflowWrap` prop in the `Text` component work here? Want to reduce the amount of inline styles 
Docs: https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-text--docs#overflow-wrap
```
import { OverflowWrap } from '../../../helpers/constants/design-system';
```
```suggestion
          overflowWrap={OverflowWrap.BreakWord}
```

Reply:
No, unfortunately it didn't work when I tried @georgewrmarshall